### PR TITLE
document regex string schemas

### PR DESCRIPTION
**Pull request by euccastro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,10 +172,35 @@ Using `:string` Schema:
 
 ```clj
 (m/validate :string "kikka")
-; => true
+;; => true
```
Comment (euccastro) [inline README.md:175]: Drive-by: other examples include double semicolons too.

**Comment by ikitommi** [comment]
Thanks!
